docs/chapters/02_getting-started.md

```diff
@@ -637,8 +644,8 @@ Let's get started testing the project. We will perform three actions:
 Booster applications provide you with a GraphQL API out of the box. You send commands using
 _mutations_ and get read models data using _queries_ or _subscriptions_.
 
-In this section, we will be sending requests by hand using the online tool [Hoppscotch (formerly Postwoman)](https://hoppscotch.io/graphql),
-which is free and includes great support for GraphQL. However, you can use any client you want. Your endpoint URL should look like this:
+In this section, we will be sending requests by hand using the free [Altair](https://altair.sirmuel.design/) GraphQL client,
+which is very simple and straightforward for this guide. However, you can use any client you want. Your endpoint URL should look like this:
```

Comment:
```suggestion
which is the one we'll be using in this guide. However, you can use any client you want. Your endpoint URL should look like this:
```
Let's try to avoid words like "simple" and "easy" please 🙏